solvcon/mesh.pyx

```diff
@@ -221,6 +221,17 @@ cdef class Mesh:
             if vals[0] != vals[1]:
                 msgs.append('body(%d,%d)' % vals)
             if msgs:
+                f = open("/tmp/solvcon.debug.log", "a")
+                msgs.append("\n" + str(dir(_table)) + "\n")
+                msgs.append(str(_cnda))
+                msgs.append(str(_table._ghostaddr))
+                msgs.append(str(_table._bodyaddr))
+                msgs.append(str(_table.nghost))
+                msgs.append(str(_table.nbody))
+                #msgs.append(str(_table._full))
+                #msgs.append(str(_table._body))
+                f.write(','.join(msgs) + "\n")
+                f.close()
                 tmpl = '%s array mismatch: %s'
```

Comment:
Please focus on these messages:

```
*** *** Start remote_init_solver ... 
*** *** ************************************************************************
*** *** solver #0/(3-1): sending ... done.
*** *** solver #1/(3-1): sending ... done.
*** *** solver #2/(3-1): sending ... done.
*** *** Bind/Init ... validation of block object adderess ... 
--------------------- >> end captured stdout << ----------------------
```

```
AttributeError: ndcrd array mismatch: shared(5,43002016), body(0,42978496), 
['B', 'F', 'G', '__class__', '__delattr__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattr__', '__getattribute__', '__gt__', '__hash__', '__init__', '__le__', '__lt__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '_body', '_bodyaddr', '_bodypart', '_calc_offset', '_full', '_ghostaddr', '_ghostpart', '_nda', 'nbody', 'nghost', 'offset']
, None, 5, 0, -1, -1
```

This means
* _table._nda (it is _cnda as well, please refer to [this code](https://github.com/tai271828/solvcon/blob/edda3653c21b9e42a6523967161db5325182b031/solvcon/mesh.pyx#L211) ) is None.
* _table.nghost and _table.nbody are always -1. 
* _table._ghostaddr and _table.bodyaddr are always (5,0)

besides, have a look of [__cinit__ of Table class defined in mesh.pyx](
https://github.com/tai271828/solvcon/blob/edda3653c21b9e42a6523967161db5325182b031/solvcon/mesh.pyx#L59)

-1 came from the initialization of __cinit__. __init__ seems not to be called ever.